ResourceStatusSystem/scripts/dirac-rss-sync.py

```diff
@@ -149,17 +149,18 @@ def initSites():
     DIRACExit( 1 )
 
   for site, elements in sites['Value'].iteritems():
+    elementType = site.split( '.' )[0]
     parameters = { 'status': elements[0],
                    'reason': 'Synchronized',
                    'name': site,
                    'dateEffective': elements[1],
                    'tokenExpiration': Datetime,
-                   'elementType': 'Site',
+                   'elementType': elementType,
```

Comment:
No. For Sites, the elementType is the "Grid" type. You mix element and elementType I guess. This is the choice done by you and Stavros.